test/e2e/case2_config_deployment_test.go

```diff
@@ -30,13 +32,13 @@ func verifyConfigPolicyDeployment(
 	By(logPrefix + "checking the number of containers in the deployment")
 
 	deploy := GetWithTimeout(
-		client, gvrDeployment, case2DeploymentName, namespace, true, 60,
+		client, gvrDeployment, case2DeploymentName, namespace, true, 300,
```

Comment:
Are these additional timeout increases still required?

Reply:
The tests are so flakey Can I try few more?